tests/integration/nm/ovs_test.py

```diff
@@ -406,3 +499,35 @@ def test_create_vlan_over_ovs_system_interface_bond_with_same_name(
                 ]
             }
         )
+
+
+@pytest.mark.tier1
+def test_modify_state_with_ovs_dup_name_ovs_bridge_first_with_ipv4_dns(
+    ovs_bridge_first_and_ovs_interface_with_same_name_ipv4,
+):
+    desired_state = {
+        Interface.KEY: [
+            {
+                Interface.NAME: ETH1,
+                Interface.TYPE: InterfaceType.ETHERNET,
+                Interface.STATE: InterfaceState.UP,
+            }
+        ]
+    }
+    libnmstate.apply(desired_state)
```

Comment:
Technically, this is missing an assertion to assert that the apply did what it was expected to do.

Reply:
Done, I've added the assert. Thanks!